e2e/notificationmenu/workflow.ts

```diff
@@ -0,0 +1,19 @@
+export const workflowWithSecrets = `direktiv_api: workflow/v1
```

Comment:
Another very minor suggestion: So far I think we use files named `utils.ts` or `utils/worflow.ts` for stuff that is imported in tests, so we could change this file name to `utils.ts` to be consistent with that.